src/test/java/org/broadinstitute/hellbender/tools/walkers/variantutils/SelectVariantsIntegrationTest.java

```diff
@@ -328,7 +328,7 @@ public void testNoGTs() throws IOException {
     @Test
     public void testSelectFromMultiAllelic() throws IOException {
         final String testFile = getToolTestDataDir() + "multi-allelic.bi-allelicInGIH.vcf";
-        final String samplesFile = getToolTestDataDir() + "GIH.samples.list";
+        final String samplesFile = getToolTestDataDir() + "GIH.samples.files";
```

Comment:
Fine for now -- create a ticket to discuss whether anything else needs to be done in `SelectVariants` in response to this new feature.